test/module-test.cc

```diff
@@ -189,18 +233,28 @@ TEST(module_test, vformat_to_n) {
   auto result = fmt::vformat_to_n(std::back_inserter(s), 1, "{}", store);
   char buffer[4] = {0};
   fmt::vformat_to_n(buffer, 3, "{:}", store);
-}
 
-TEST(module_test, print) {
-  EXPECT_WRITE(stdout, fmt::print("{}µ", 42), "42µ");
-  EXPECT_WRITE(stderr, fmt::print(stderr, "{}µ", 4.2), "4.2µ");
+  auto wstore = fmt::make_wformat_args(12345);
+  std::wstring w;
+  auto wresult = fmt::vformat_to_n(std::back_inserter(w), 1,
+                                   fmt::to_string_view(L"{}"), wstore);
+  wchar_t wbuffer[4] = {0};
+  fmt::vformat_to_n(wbuffer, 3, fmt::to_string_view(L"{:}"), wstore);
 }
 
 std::string as_string(std::wstring_view text) {
   return {reinterpret_cast<const char*>(text.data()),
           text.size() * sizeof(text[0])};
 }
 
+TEST(module_test, print) {
+  EXPECT_WRITE(stdout, fmt::print("{}µ", 42), "42µ");
+  EXPECT_WRITE(stderr, fmt::print(stderr, "{}µ", 4.2), "4.2µ");
+
+  EXPECT_WRITE(stdout, fmt::print(L"{}µ", 42), as_string(L"42µ"));
+  EXPECT_WRITE(stderr, fmt::print(stderr, L"{}µ", 4.2), as_string(L"4.2µ"));
```

Comment:
These can be problematic because IIRC they may fail if the stream is not in wide mode. I suggest only checking that this compiles without actually running, i.e. wrap in `if (false)`.

Reply:
Ok, will do.